guides/common/modules/con_user-groups.adoc

```diff
@@ -3,6 +3,6 @@
 
 With {Project}, you can assign permissions to groups of users.
 You can also create user groups as collections of other user groups.
-If using an external authentication source, you can map {Project} user groups to external user groups as described in xref:Configuring_External_User_Groups_{context}[].
+If using an external authentication source, you can map {Project} user groups to external user groups as described in link:{InstallingServerDocURL}Configuring_External_User_Groups_{project-context}[Configuring external user groups].
```

Comment:
```suggestion
If using an external authentication source, you can map {Project} user groups to external user groups as described in {InstallingServerDocURL}Configuring_External_User_Groups_{project-context}[Configuring external user groups] in _{InstallingServerDocTitle}_.
```

We always omit the `link:` prefix. Frr a list of titles, see https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-titles.adoc

Reply:
thank you, I have removed the "link:" parts from all references. I have also added names of titles in italics